components/hook-exit/node/index.test.mjs

```diff
@@ -0,0 +1,13 @@
+import process from "node:process";
+import { assertDeepEqual } from "../../__fixture__.mjs";
+import { testHookAsync } from "../../hook-fixture/index.mjs?env=test";
+import * as HookError from "./index.mjs?env=test";
```

Comment:
This should probably be called `HookExit`.

Reply:
done in 990a60a4bf6e041822cb43e669b1f635b089d15b